### ISSUE TITLE
Audit Spotlight wiki

### ISSUE THREAD

**Issue by camillevilla** [projectblacklight/spotlight#2023]
In preparation for Spotlight 2.0, let's re-organize our wiki and analyze which areas are out of date.

**Comment by anarchivist** [comment]
* Split docs up between team, assign specific pages to team members
* Flag things that are out of date
* Triage/prioritize work for correction
* Correct those things (or mark as deprecated?)
* If time, consider reorganizing if it makes sense / helps with clarity

**Comment by mejackreed** [comment]
- [ ] Home - @mejackreed
 - [ ] Installation
 - [ ] Blacklight
 - [ ] Page widgets
 - [ ] Resource scenarios
 - [ ] Image versions
 - [ ] Queueing backends

**Comment by camillevilla** [comment]
@ggeisler I agree that the stub pages aren't likely to get filled out soon. I'm excited about the new documentation exhibit, but I'm worried about folks getting confused about what comes in Spotlight out-of-the-box vs. Stanford customizations. 

Perhaps we could export the Stanford documentation exhibit and import it into the Spotlight test app? I'm ok with waiting until the documentation exhibit is published for us to make a decision though. 

In the mean time I would be interested in:
1. Removing the stub sections under "Building an exhibit"
2. Retaining [Configuration settings](https://github.com/projectblacklight/spotlight/wiki/Configuration-settings) and adding a brief section about Languages that links to [Translations](https://github.com/projectblacklight/spotlight/wiki/Translations)

**Comment by aeschylus** [comment]
I don't see anything in the `resource scenarios` page that is inaccurate or out-of-date as far as I know, but might add a section about "indexing" OCR items that discusses adding a IIIF search service endpoint to manifests, or rather, importing manifests that happen to link to a service. Is this relevant here?
